### PR TITLE
Add `setScale` in PlayerAPI & EntityAPI in LLSE

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: zh-Hans/LLSEPluginDevelopment/GameAPI/Player.md [hunk 1]
```diff
@@ -416,6 +416,16 @@
     pl:setFire(20,true)
     ```
 
+#### 比例播放器
```
Comment (Futrime) [inline zh-Hans/LLSEPluginDevelopment/GameAPI/Player.md:419]: Here should be "缩放玩家"

File: zh-Hans/LLSEPluginDevelopment/GameAPI/Player.md [hunk 2]
```diff
@@ -416,6 +416,16 @@
...
+
+`pl.setScale(scale)`
+
+- 参数: 
+  - scale : `Integer`  
+    新播放器尺寸
```
Comment (futrime) [inline zh-Hans/LLSEPluginDevelopment/GameAPI/Player.md]: The word "player" should be translated into "玩家"

File: zh-Hans/LLSEPluginDevelopment/GameAPI/Player.md (hunk 1 above)
Comment (futrime) [inline zh-Hans/LLSEPluginDevelopment/GameAPI/Player.md]: Here should be "缩放玩家"

**Comment by Futrime** [comment]
Modify the content as I commented. Then your PR will be merged as soon as your code is merged to the LiteLoaderBDS repo.

**Review by futrime — CHANGES_REQUESTED** [review]
Please modify the content according to my comments.

**Comment by ghost** [comment]
I don't know chinese and use translator for translating on chinese. Can translate docs only english in future?

**Comment by futrime** [comment]
@shishkevichd That's OK. But you should still keep a copy of the English content at the same place of the Simplified Chinese docs. We will help you translate it.

**Comment by ghost** [comment]
> @shishkevichd That's OK. But you should still keep a copy of the English content at the same place of the Simplified Chinese docs. We will help you translate it.

Okay, i will do so.

**Review by futrime — APPROVED** [review]

**Pull request merged**